### PR TITLE
Adding functionality to view individual post and also delete it.

### PR DESCRIPTION
**Pull request by saadyousafarbi**
This **PR** is responsible for viewing individual `posts` and also provides user with the ability to `delete` specific selected `post`. User will be provided with a `link` to each of his/her `posts'` detail on the `main page`. The selected `posts` page will include a `delete button` that would delete that particular `post` from the user's blog profile.

**Main Blog Page**

main_page_blog

**View/Delete Post Page**

delete_post

**Add New Post Page**

add_post

### CODE REVIEW

File: src/components/NewPost.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+class NewPost extends Component {
+    static ContextType = {
+        router: PropTypes.object
+    };
+
+    onSubmit(props) {
+        this.props.createPost(props)
+        .then(() => {
+            this.props.router.push('/');
```
Comment (verdan) [inline src/components/NewPost.js]: sweet!!

File: src/components/ShowPost.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                <h4> Categories: {this.props.post.categories} </h4>
+                <p> Content: {this.props.post.content} </p>
+            </div>
+            </div>
+        )
+    }
+}
+
+function mapStateToProps(state) {
+    return {  post : state.posts.selectedPost };
```
Comment (verdan) [inline src/components/ShowPost.js]: please remove extra spacing.

File: public/index.html [hunk 3]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```
Comment (zubair-arbi) [inline public/index.html]: use local css.

**Comment by saadyousafarbi** [comment]
Squashed from b1ad81fde69a8bb3ef0dc74d72936cc7dd883620

**Pull request merged**